harness/tests/determined/common/experimental/test_trial.py

```diff
@@ -70,3 +70,42 @@ def test_trial_logs_invalid_timestamps(
     with pytest.raises(ValueError):
         list(trialref.logs(timestamp_before=timestamp, timestamp_after=None))
         list(trialref.logs(timestamp_before=None, timestamp_after=timestamp))
+
+
+@pytest.mark.parametrize(
+    "name,valid_timestamp",
+    [
+        ("no_microseconds", "2021-10-26T23:17:12Z"),
+        ("microseconds", "2021-10-26T23:17:12.0000Z"),
+    ],
+)
+@responses.activate
+def test_trial_logs_valid_timestamps(
```

Comment:
nit: `test_trial_logs_accepts_valid_timestamps` is slightly clearer to a debugger

Reply:
done